### ISSUE TITLE
Improve schematic view in the examples

### ISSUE THREAD

**Issue by failiz** [fritzing/fritzing-app#3962]
## Problem
The examples do not have very good schematics. They do not use power symbols or net labels, which makes the difficult to read. And they do not encourage learners to use them. This will also give more visibility to power symbols and the net labels.

Some examples as an illustration if the issue:
![image](https://user-images.githubusercontent.com/5708642/179360670-8c590f2e-e353-46a8-81e3-dcb57f16edd1.png)
![image](https://user-images.githubusercontent.com/5708642/179360679-b8d8500e-8c51-4f43-b46f-f3a818621488.png)
![image](https://user-images.githubusercontent.com/5708642/179360688-858817d7-e497-40b0-a5ab-748619208c6b.png)
![image](https://user-images.githubusercontent.com/5708642/179360712-f0ec9d93-0cf1-49a9-9374-aec576c76a27.png)

## Proposed Solution
Improve the schematic view of the examples by using power symbols, net labels, and standard electric engineering practices.
We should also consider if we want to keep wires with different colors in the schematic view. I find that very weird.
See for example my NAND gate schematic (I will add this example to the simulator examples)
![image](https://user-images.githubusercontent.com/5708642/179360798-64451339-3dec-42da-ba4a-c6c3894f4ec0.png)

**Comment by vanepp** [comment]
"We should also consider if we want to keep wires with different colors in the schematic view. I find that very weird." 

Me too, but some sketches I have seen from users use colors in schematic, so at least some of the user base appears to use them. I'd say if we are considering this we should solicit input from the user community perhaps via the forum (although forum usage has dropped a lot from past years, I don't know of a better place either!)

**Comment by failiz** [comment]
I do not think we should remove the functionality, it could sometimes be helpful to illustrate how schematics work. But I think the colors should be removed from the examples.

**Comment by failiz** [comment]
The examples of the simulator (and probably all of the others too) are still using the old potentiometers (the pots with the old, and wrong, symbol). And the resistor and the diode still show the "0.25" (max. power of the resistor) and "0.003" (max current of the LED). We should update the parts in the examples to the new versions of the parts to avoid this.

**Comment by vanepp** [comment]
Interesting! I haven't really looked at the examples but now that I do I see what you mean. For example the first one I tried: arduino/output/blink has parts in core parts in the temp parts bin (when I think they should be using the ones in core!) 

![capture](https://github.com/fritzing/fritzing-app/assets/16514277/2e6341e4-8fe2-490e-b514-72f99fe8af5c)

I think we would be better off modifying the example .fzz files to use core parts where they exist. I'm not sure we need to do this for the 1.0 release, but we should I think add it to the todo list. Indeed in the simulator example for the pot it is using the old style pot rather than the current one (possibly because we didn't obsolete the old one?) 

![capture1](https://github.com/fritzing/fritzing-app/assets/16514277/744ba283-e6c7-4140-930f-7d4e7c037c8c)

where the one in core parts (this is on 1.0!) is the new pot

![capture2](https://github.com/fritzing/fritzing-app/assets/16514277/12222f90-208b-4599-89ef-ede3701375d0)

This should be a relatively easy update I expect, just modify the .fzz files in the examples directory.

**Comment by KjellMorgenstern** [comment]
@vanepp Are you already using 1.0.0RC2 ? While fixing or cleaning up some of the examples, we had used a dev build with not the official fritzing-parts branch. In release candidate 1 the examples therefore included copies of the parts. For RC2 this should be fixed, but i might have missed one?

**Comment by vanepp** [comment]
I think I should be on rc2 on Windows 10

Version 1.0.0 
(rcCD-4149-0-19b5f937 2023-05-25) 64 [Qt 6.4.3]

edit:

On 9.10 it works as expected:

![capture](https://github.com/fritzing/fritzing-app/assets/16514277/4f15ba55-6882-40b0-b4db-b0c026c780d3)

(no temp parts directory) but on what I think is rc2 it has a temp directory. That is the only one I see so far so it may be a single missed fzz file (I was wondering if it was already broken in 0.9.10 and just hadn't been noticed yet!) 

![capture1](https://github.com/fritzing/fritzing-app/assets/16514277/e3c0f612-82b2-456e-8691-3916ea66029c)

I'll continue trying the examples and see if I find more with a temp parts bin.

**Comment by vanepp** [comment]
edit2:

Since I'm here :-) 

![capture2](https://github.com/fritzing/fritzing-app/assets/16514277/95b56bd8-730e-49f3-87da-86a28076f09a)

has the old pot (I suspect that needs a manual update to get the new one!) another one with a temp parts bin

![capture3](https://github.com/fritzing/fritzing-app/assets/16514277/08af444f-d05a-42d0-a88f-7ea2ff51025d)

old style pot

![capture4](https://github.com/fritzing/fritzing-app/assets/16514277/3efe04f0-9177-4505-9beb-43c2fbfd6f82)

old style pot

![capture5](https://github.com/fritzing/fritzing-app/assets/16514277/36b53fce-e4a5-4157-ba49-3bdd7e37da28)

old style pot


![capture6](https://github.com/fritzing/fritzing-app/assets/16514277/e614c08d-29d3-4ed8-9ae1-0accdd3b9b08)

old style pot


![capture7](https://github.com/fritzing/fritzing-app/assets/16514277/dcf54a3b-bf33-4024-8bbf-7fb7c052eaa3)

old style pot

![capture8](https://github.com/fritzing/fritzing-app/assets/16514277/6e11a304-e88a-4c59-9579-7df0c2a9c993)

old style pot

![capture9](https://github.com/fritzing/fritzing-app/assets/16514277/06e4f3ac-bb56-4b03-b04d-82d1c235ce7d)

old style fsr (I think anyway!) 

![capture10](https://github.com/fritzing/fritzing-app/assets/16514277/f4122f40-fa77-4e19-a2f2-778b0cabe4af)

temp parts bin. I think the parts should all be in core (the next one along on perf board uses the same parts with no temp parts bin!) 

![capture11](https://github.com/fritzing/fritzing-app/assets/16514277/a356f5bd-bf48-43b7-8ec8-5ac996725d06)

old style pot


![capture12](https://github.com/fritzing/fritzing-app/assets/16514277/6ec67451-3e8b-446e-a5d4-ab44f7afcd8f)

I didn't go through the starter kit or creator kit as AFAIK they are old and probably don't need updating.

**Comment by KjellMorgenstern** [comment]
I think these were already solved on 1.0.1 or 1.0.2, however we just checked the examples on a local development build. Therefor I just mark it as solved, to be closed with the 1.0.3 release.

**Comment by failiz** [comment]
I do not agree that this issue has been solved. The original issue (bad quality schematic diagrams is still present in most of the examples). I think we have replaced the old pots and force sensor resistors with the new parts, but schematics still do not use GND, power symbols, and net labels and they have wires with different colors.  

See some random examples: 
![image](https://github.com/fritzing/fritzing-app/assets/5708642/10a5e508-5923-417a-913b-42dd418d85b5)
![image](https://github.com/fritzing/fritzing-app/assets/5708642/0dd8f0db-8f07-4a81-9bf9-87533aebcad5)
![image](https://github.com/fritzing/fritzing-app/assets/5708642/7661d0a4-7139-4f32-a375-761def2379f6)
![image](https://github.com/fritzing/fritzing-app/assets/5708642/2a370f86-0ec1-45a9-bcef-cf5a5e6354d2)
